pydicom/dataset.py

```diff
@@ -149,6 +149,13 @@ def __init__(self, *args, **kwargs):
         dict.__init__(self, *args)
         self.is_decompressed = False
 
+        # these will be set if the dataset is read from a file
+        self.read_little_endian = None
```

Comment:
If these are public attributes then you should update the class docstring, if private then maybe use `self._read_x` and just add an inline comment to explain what they're for.

Reply:
I have been unsure about these. Technically, they should be internal to pydicom (similar to package internal in Java). If I make them private, I have to use a private member from other pydicom classes, which is not nice; but I also don't want to make them documented, as they should not be used from outside. Not sure what is the best her - what is the Pythonic way here?

Reply:
Yeah I have the same issue...

Maybe leave it out of the docstring and just use inline comments then?

Reply:
Agreed, will do this.

Reply:
Done.